### PR TITLE
Add type size when exporting postgres

### PR DESCRIPTION
**Pull request by rudcode**

### CODE REVIEW

File: src/data/datatypes.js [hunk 1]
```diff
@@ -857,9 +857,8 @@ const postgresTypesBase = {
       return field.default.length <= field.size;
     },
     hasCheck: true,
-    isSized: true,
+    isSized: false,
     hasPrecision: false,
-    defaultSize: 65535,
```
Comment (belyas) [inline src/data/datatypes.js:862]: how these changes would impact other database engines?

File: src/utils/exportSQL/postgres.js [hunk 2]
```diff
@@ -31,7 +31,9 @@ export function toPostgres(diagram) {
             (field) =>
               `${exportFieldComment(field.comment)}\t"${
                 field.name
-              }" ${field.type}${field.isArray ? " ARRAY" : ""}${field.notNull ? " NOT NULL" : ""}${field.unique ? " UNIQUE" : ""}${
+              }" ${field.type}${
+                field.size !== undefined && field.size !== "" ? "(" + field.size + ")" : ""
```
Comment (belyas) [inline src/utils/exportSQL/postgres.js:35]:
```suggestion
                field.size ? "(" + field.size + ")" : ""
```
isn't this enough? or we need to test against undefined/empty for some reason?

**Review by belyas — CHANGES_REQUESTED** [review]

File: src/utils/exportSQL/postgres.js (hunk 2 above)
Comment (rudcode) [reply src/utils/exportSQL/postgres.js:35]: Hi, I just copy this from https://github.com/drawdb-io/drawdb/blob/main/src/utils/exportSQL/mysql.js#L12

File: src/data/datatypes.js (hunk 1 above)
Comment (rudcode) [reply src/data/datatypes.js:862]: This shouldn't impact any other database, because each database have their own {database}TypesBase.

File: src/utils/exportSQL/postgres.js (hunk 2 above)
Comment (1ilit) [reply src/utils/exportSQL/postgres.js:35]: Not all fields have a size so yeah

**Review by 1ilit — APPROVED** [review]
